src/shell/shell.c

```diff
@@ -1114,7 +1114,17 @@ int main (int argc, char *argv[])
             shell_die (1, "failed to initialize taskid=%d", i);
 
         if (shell_task_start (task, shell.r, task_completion_cb, &shell) < 0) {
-            shell_die (1, "task %d: start failed: %s: %s",
+            int ec = 1;
+            /* bash standard, 126 for permission/access denied, 127 for
+             * command not found.  68 (EX_NOHOST) for No route to host.
+             */
+            if (errno == EPERM || errno == EACCES)
+                ec = 126;
+            else if (errno == ENOENT)
+                ec = 127;
+            else if (errno == EHOSTUNREACH)
```

Comment:
Since the shell is only launching local tasks, can we ever get `EHOSTUNREACH` from shell_task_start?
(I'm probably just missing a corner case, since you probably saw this errno during testing)

Reply:
> (I'm probably just missing a corner case, since you probably saw this errno during testing)

You think too highly of me :-)  I blindly copied behavior from other parts of Flux.  You're absolutely right, the shell only does local tasks.